client/src/app/site/deployment-center/deployment-center-setup/step-complete/step-complete.component.ts

```diff
@@ -149,10 +149,12 @@ export class StepCompleteComponent {
         value: buildSettings.vstsAccount,
       });
 
-      returnSummaryItems.push({
-        label: this._translateService.instant(PortalResources.project),
-        value: buildSettings.vstsProject,
-      });
+      if (!wizValues.buildSettings.createNewVsoAccount) {
```

Comment:
Should we instead show the default project name? This would give idea to the user where the pipeline will be configured.

Reply:
By default there will be one project only for new organization.